curriculum/challenges/english/08-coding-interview-prep/rosetta-code/run-length-encoding.md

```diff
@@ -0,0 +1,128 @@
+---
+id: 5eb257ba998e23057d209754
+title: Run-length Encoding
+challengeType: 5
+---
+
+## Description
+<section id='description'>
+Given a string containing uppercase characters (A-Z), compress repeated 'runs' of the same character by storing the length of that run, and provide a function to reverse the compression.
+</section>
```

Comment:
We can reorganize this section a bit so it more closely resembles the original Rosetta Code description:

````
<section id='description'>
Given a string containing uppercase characters (A-Z), compress repeated 'runs' of the same character by storing the length of that run, and provide a function to reverse the compression.

Example:<br>
Original: `ABBBCDDDD`<br>
Encoded: `[[1, 'A'], [3, 'B'], [1, 'C'], [4, 'D']]`
</section>
````